openedx_events/learning/signals.py

```diff
@@ -7,3 +7,84 @@
 They also must comply with the payload definition specified in
 docs/decisions/0003-events-payload.rst
 """
+
+from openedx_events.learning.data import (
+    CertificateData,
+    CohortData,
+    CourseEnrollmentData,
+    RegistrationFormData,
+    StudentData,
+)
+from openedx_events.tooling import OpenEdxPublicSignal
+
+STUDENT_REGISTRATION_COMPLETED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.student.registration.completed.v1",
+    data={
+        "user": StudentData,
```

Comment:
This naming mismatch is throwing me off a bit.  Does it make more sense for the key to be `student` instead of `user`?